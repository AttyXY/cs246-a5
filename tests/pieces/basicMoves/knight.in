setup
+ k a1
+ K h8
+ p b7
+ N f3
done
game human human
move f3 g1
move a1 a2
move g1 f3 
move a2 a1
move f3 e1
move a1 a2
move e1 f3
move a2 a1
move f3 h2
move a1 a2
move h2 f3
move a2 a1
move f3 h4
move a1 a2
move h4 f3
quit
quit
