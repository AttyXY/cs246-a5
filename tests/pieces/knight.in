game human human
move b1 a3
move b8 c6
move g1 f3
move g8 h6
move a3 c4
move c6 a5 
move f3 d4 
move h6 f5
move c4 d6
move a5 g3
move d6 b7
move g3 f1
