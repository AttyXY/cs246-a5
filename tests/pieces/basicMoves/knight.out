WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

WELCOME TO SETUP MODE!
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- + [PieceType] [Coordinate] (to add piece)
- - [Coordinate] (to remove piece)
- = [Colour] (to set turn)
- done (to exit setup mode)

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  --------
1  k-------
   abcdefgh

PLEASE ENTER THE FOLLOWING COMMANDS TO BEGIN:
- game [player1] [player2]

STARTING GAME!

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k-----N-
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  k-------
1  ------N-
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  k-------
1  --------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  --------
1  k---N---
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  k-------
1  ----N---
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  k-------
1  --------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  -------N
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  --------
2  k------N
1  --------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  k-------
1  --------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  -------N
3  --------
2  --------
1  k-------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  -------N
3  --------
2  k-------
1  --------
   abcdefgh

8  -------K
7  -p------
6  --------
5  --------
4  --------
3  -----N--
2  k-------
1  --------
   abcdefgh


PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0
Black: 0
